lib/consolidate.js

```diff
@@ -1760,6 +1761,33 @@ exports.squirrelly.render = function(str, options, cb) {
     }
   });
 };
+/**
+ * Twing support.
+ */
+
+exports.twing = fromStringRenderer('twing');
+
+/**
+ * Twing string support.
+ */ 
+
+exports.twing.render = function(str, options, cb) {
+  if (!semver.satisfies(process.version, '>=6.0.0')) {
+    return promisify(cb, function(cb) {
```

Comment:
but now it need node 8.0.0 ... hope this is okay ... but since we can use semver now the packages should be able to define the supported node versions independently